go.mod

```diff
@@ -17,13 +17,13 @@ require (
 	github.com/stretchr/testify v1.8.4
 	google.golang.org/grpc v1.57.0
 	google.golang.org/protobuf v1.31.0
-	k8s.io/api v0.28.0-rc.0
```

Comment:
github.com/kubernetes-csi/csi-lib-utils v0.14.0 we should update this to v0.15.0

Reply:
@msau42 @xing-yang should we use `csi-lib-utils:v0.15.0-rc0` instead? It does not have the changes for Optel package.

Reply:
I think we can stay on v0.14.0 which should be equivalent

Reply:
Ok sounds good. Resolving the comment as no further changes are needed